### PR TITLE
Added some missing functions

### PR DESCRIPTION
**Pull request by vshymanskyy**

### CODE REVIEW

File: lib/blynk/BlynkLib.py [hunk 1]
```diff
@@ -355,6 +360,10 @@ def tweet(self, msg):
     def email(self, to, subject, body):
         if self.state == AUTHENTICATED:
             self._send(self._format_msg(MSG_EMAIL, to, subject, body))
+            
+    def virtualWrite(self, pin, val): # TODO: *args?
```
Comment (dpgeorge) [inline lib/blynk/BlynkLib.py]: Should probably be `virtual_write`.

Comment (danicampora) [inline lib/blynk/BlynkLib.py]: @dpgeorge Yes, `virtual_write` is the Pythonic form.

Comment (danicampora) [inline lib/blynk/BlynkLib.py]: @vshymanskyy why *args? Can you write multiple values at once (a list?) to a virtual pin?

Comment (vshymanskyy) [inline lib/blynk/BlynkLib.py]: fixed.
@danicampora , yes this is supported by the protocol. (when you receive write handler, there is also a list of args).
However, we don't need to support it immediately.

**Comment by danicampora** [comment]
@vshymanskyy Thanks! will merge ASAP (shipping is taking over my life...)

**Comment by danicampora** [comment]
@vshymanskyy Merged in https://github.com/wipy/wipy/commit/d3d9535c69662477406b31c03b00a5e7c0d23f5a

**Comment by danicampora** [comment]
Thanks!!

**Pull request closed without merging**